README.md

```diff
@@ -1,5 +1,7 @@
 # Compute@Edge static content starter kit for Rust
 
+[![Deploy to Fastly](https://deploy.edgecompute.app/button)](https://deploy.edgecompute.app/deploy)
+
```

Comment:
Can you also double check the rest of the README still reflects the actual code. Thanks!